modules/ROOT/pages/dataweave-variables.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
 
-DataWeave is a functional programming language where variables behave just like functions.
+DataWeave is a functional programming language in which variables behave just like functions. DataWeave eagerly evaluates variables, the same way it evaluates arguments to functions. In addition, DataWeave variables are immutable. 
```

Comment:
eagerly?

Reply:
I wonder if we could have one sentence instead of three: that in DW, variables not only behave and are evaluated in the same way as functions but also are immutable.

Reply:
That sounds better. However, our team is likely to add more about immutability, so I want to keep that sentence separate. It's a stop-gap at the moment.